packages/msgraph/src/Adaptor.js

```diff
@@ -254,9 +254,9 @@ export function getFile(pathOrId, options, callback = s => s) {
 
     const auth = getAuth(accessToken);
 
-    return request(url, { ...auth }).then(response =>
-      handleResponse(response, state, callback)
-    );
+    const response = await request(url, { ...auth, parseAs });
```

Comment:
I would ideally like to touch this function as little as possible, can we revert most of the refactoring here and just do:
```
 return request(url, { ...auth, parseAs: 'text' }).then(...)
```

Reply:
@josephjclark `getFile()` can return metadata which is `json` or can return file content which in this case `text`
So if we pass `parseAs: text` the all test that returns metadata will fail unless in those test i pass 
`parseAs: json`. But i have thought of a quick fix which can be `parseAs: metadata ? 'json' : 'text',`
So the response will be like this.
```
const response = await request(url, {
      ...auth,
      parseAs: metadata ? 'json' : 'text',
    });
```

This way the function still behave the same and we don't have to update the tests

Reply:
https://github.com/OpenFn/adaptors/pull/345/commits/cb97fff6d74e50192a7fa98fcf9951b3d54850ea